libraries/stdlib/common/src/generated/_Ranges.kt

```diff
@@ -979,7 +981,8 @@ public infix fun LongProgression.step(step: Long): LongProgression {
 
 /**
  * Returns a progression that goes over the same range with the given step.
- * @sample samples.ranges.Ranges.stepChar
+ * 
+ * @sample samples.ranges.Ranges.stepInt
```

Comment:
Why stepInt sample is used for CharProgression?

Reply:
fixed now.